mobile/src/main/java/rs/readahead/washington/mobile/views/dialog/reports/edit/EditTellaServerFragment.kt

```diff
@@ -145,4 +153,14 @@ class EditTellaServerFragment :
         return server
     }
 
+    private fun setNoTimeOut(enableNoTimeout: Boolean) {
```

Comment:
The method name and parameter can be made more descriptive to better describe their purpose.Maybe something like "updateTimeoutSetting"
let is assign the timeout value once after the conditional check to reduce redundancy.